### PR TITLE
Updates Firmware Bugs Resolved in 10.01 Release

### PR DESCRIPTION
**Pull request by hm-ti**
AM64X: Update 10.01 Issue Tracker
Issue Tracker Update containing details of firmware Bugs Resolved and Bugs Open.

### CODE REVIEW

**Review by cshilwant — CHANGES_REQUESTED** [review]
- Drop commit 6082cdafd0070fbe1c51218bfcc93af23a71798c

- 5ccd5f8f1bdafa64acfb6cb99c68b4e3eadac797 and f7916b7dc576658aae6b3826dbe199870b6dda77 should go as a single commit

File: source/devices/AM64X/linux/Release_Specific_Release_Notes.rst [hunk 1]
```diff
@@ -219,6 +219,14 @@ Issues Resolved
...
    "SYSFW-6432","Set device API doesn't return Error when PD is in transition state"
    "EXT_SITMPUSW-32", "SDK Makefile: Fix logic to pick key directory for mkimage"
+   "PINDSW-7087","Ethernet: IET: Frame preemption MAC verify fails"
+   "PINDSW-8229","Ethernet: FDB: FDB entries missing with link up/down activity"
+   "PINDSW-8187","Ethernet: FDB: MC frame is not received when only specific port bit is set in FID_C2 (Applicable only for DUAL MAC mode)"
+   "PINDSW-8086","Ethernet: FDB: Port blocked/disabled use case for special frame"
+   "PINDSW-8022","Ethernet: TAPRIO: Moved base time usage into the firmware"
+   "PINDSW-7988","Ethernet: CT: Cut-thru can not be verified (Missing PA statistics)"
+   "PINDSW-8273","Ethernet: 10M: Link Status Valid flag is not set in case of 100M/1G"
+   "PINDSW-8073","ICSSG Switch: PTP: Interrupt is coming on the other port"
```
Comment (cshilwant) [inline source/devices/AM64X/linux/Release_Specific_Release_Notes.rst:229]: No SIR for PINDSW as well? 

@praneethbajjuri

**Comment by cshilwant** [comment]
Made the following change in PR subject,

s/Reelase/Release

**Comment by hm-ti** [comment]
Created a new PR for resolving the issue with commits. 
https://github.com/TexasInstruments/processor-sdk-doc/pull/148

**Pull request closed without merging**